docs/UserGuide.md

```diff
@@ -258,34 +258,35 @@ It consists of three main parts:
 * [Code Panel](#code-panel)
 
 ### Tool Bar
-The `Tool Bar` at the top provides a set of filters that control the chart panel. From right to left, the filters are:
-- Sort : Users can sort by:
-    - Total Contribution : The amount of lines, written by the author, in the repository.
-    - Variance : The variance of contribution of all commits.
-        - It indicates the code consistency of an author.
-        - It helps to determine whether the author has been contributing regularly or procrastinating.
-        - This can be useful for instructors of student projects, e.g. [Addressbook](https://github.com/se-edu/addressbook-level4).
-    - Author Name : The Author's GitHub ID or Display Name configured in the `CSV Config File`.
-    - Team Name : The name of the organization of the repository.
-- Interval : Interval refers to amount of time one single ramp represents.
-    - Users can choose two modes of time intervals:
-        - daily
-        - weekly
-- Period : The time period that the ramp charts display.
-- Group : Checkbox.
-    - If checked, authors that have contributed to the same repository will be displayed next to each other.
-    - This overrides the sort function.
-- Search : Filters the author and repository by keywords.
-    - Multiple keywords can be used, separated by spaces.
-    - The keywords are logically connected with OR operators.
-- Bookmark : The star icon at the top right corner.
-    - Clicking on it will generates a link to the report with all the tooltip settings.
-    - This link will be copied to user’s clipboard.
+The `Tool Bar` at the top provides a set of filters that control the chart panel.
+- `Search` : filters the author and repository by keywords.
+  - Multiple keywords/terms can be used, separated by spaces.
+  - Only entries that contain _any_ (not necessarily _all_) of the search terms will be displayed.
+- `Sort by` :
+  - `Total Contribution` : the amount of lines, written by the author, in the repository.
+  - `Variance` : based on the number of lines that the author has contributed to the repository on each interval.
+    - It indicates the consistency on the number of lines that user has contributed towards the repository over the time.
```

Comment:
Rather, state _exactly_ what we do so that the user can ignore that feature if the formula is not good enough for her purpose. If we change to a different formula later, we can update the description. Given the context of our tool, we should always report data/features as they are, and leave to the user to draw their own inferences.

Reply:
I've changed the message to as followed:

  - `Variance` : calculated based on the number of lines that the author has contributed to the repository daily.
    - Sorts the authors in the order of from low to high variance value.
    - A low variance value indicates that the number of lines that an author has contributed daily are consistent.

Would this be precise enough?

Reply:
🤔 Still not precise enough. What exactly is _variance_ here? e.g., the difference between the highest and the lowest contribution?

Reply:
Or are we using the standard defintion of the variance (as used in statistics)?

Reply:
> Or are we using the standard definition of the variance (as used in statistics)?

Oh I see, I thought the name itself would be pretty much explanatory. Should I include this line below?
* This is calculated using the [variance formula](https://en.wikipedia.org/wiki/Variance) defined in the probability theory and statistics.

Reply:
Something like this is enough?
`Variance` : the [variance]() of the number of lines that the author has contributed to the repository daily, sorted from low to high.

Reply:
Done, along with other changes